kebechet/config.py

```diff
@@ -146,7 +146,7 @@ def run_url(cls, url: str, service: str, parsed_payload: dict, tls_verify: bool)
                 )
                 continue
             _LOGGER.info(f"Running manager %r for %r", manager_name, slug)
-            manager_configuration = manager.pop("configuration", {})
```

Comment:
Wasn't this more safer? `manager.pop("configuration") or {}` gives an error when manager isn't there?

Reply:
Right, it should be `get` method. Thanks for spotting this :+1: